src/main/java/com/launchdarkly/client/StreamProcessor.java

```diff
@@ -119,8 +128,9 @@ public void close() throws IOException {
     }
   }
 
-  boolean initialized() {
-    return store.initialized();
+  @Override
+  public boolean initialized() {
+    return initialized && store.initialized();
```

Comment:
The go client no longer checks if the store is initialized, which is probably the desired behavior here.